modules/seqexec/server/src/main/scala/seqexec/server/InstrumentControllerSim.scala

```diff
@@ -87,27 +88,27 @@ object InstrumentControllerSim {
       observeTic(stop = false, abort = false, pause = false, totalTime, useTimeout.option(totalTime + 2 * tic))
     }
 
-    def applyConfig[C: Show](config: C): F[Unit] = Sync[F].delay {
+    def applyConfig[C: Show](config: C): F[Unit] = F.delay {
       Log.info(s"Simulate applying $name configuration ${config.show}")
       Thread.sleep(ConfigurationDelay.toMilliseconds.toLong)
     }
 
-    def stopObserve: F[Unit] = Sync[F].delay {
+    def stopObserve: F[Unit] = F.delay {
       Log.info(s"Simulate stopping $name exposure")
-      Thread.sleep(1500)
+      Thread.sleep(StopObserveDelay.toMilliseconds.toLong)
```

Comment:
Why not use your `Timer` instance here and elsewhere?

Reply:
Yeah I need to refactor this. I'll do it in another PR, this is a known issue but I want to tackl this as part of the N&S simulation support part